master/buildbot/process/log.py

```diff
@@ -176,12 +174,18 @@ def wholeLines(lines):
             return lbf
 
     def addStdout(self, text):
+        if not isinstance(text, str):
+            text = self.decoder(text)
         return self._getLbf('o').append(text)
 
     def addStderr(self, text):
+        if not isinstance(text, str):
+            text = self.decoder(text)
```

Comment:
the coverage says this is never executed. Do we really need it? don't we have some autodecode before?

I am pretty sure bytes are used on the network link, so maybe it is already converted before.

Reply:
This will be covered by a subsequent PR where some master steps were changed to new style. This is how I noticed the problem at first. I just thought it makes sense to have a separate PR to not mix everything.